packer.yaml.jinja2

```diff
@@ -12,6 +12,9 @@ provisioners:
   - type: file
     source: ./files.tar
     destination: /tmp/
+
+  ### Linux only pre-script steps
```

Comment:
Can you add a comment to explain what these steps do and why they are only needed for linux?